static/app/views/lastKnownRouteContextProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import {createContext, useContext} from 'react';
+
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import usePrevious from 'sentry/utils/usePrevious';
+import {useRoutes} from 'sentry/utils/useRoutes';
+
+interface Props {
+  children?: React.ReactNode;
```

Comment:
```suggestion
  children: React.ReactNode;
```
we always want the children for this provider